tests/test_BT_piecewise.py

```diff
@@ -0,0 +1,560 @@
+from pint.models import get_model
+import pint.toa
+import numpy as np
+import pint.fitter
+import astropy.units as u
+from pint import ls
+from copy import deepcopy
+import pint.residuals
+from astropy.time import Time
+import pint.models.stand_alone_psr_binaries.BT_piecewise as BTpiecewise
+import matplotlib.pyplot as plt
+import unittest
+from io import StringIO
+from pylab import *
+import pytest
+import pint.residuals
+import pint.simulation
+
+
+@pytest.fixture
+def build_model():
```

Comment:
Because you use fixtures like `def test_thing(fixture):` ... `y = fixture.attribute`, it makes sense to use a noun-like name; here `model_no_pieces` would make sense, so you'd have `def test_something(model_no_pieces):`.

Reply:
Addressed this only once (other functions like: build_with_two_pieces could receive a similar treatment)